docs/389ds/FAQ/bdb-deprecation.md

```diff
@@ -0,0 +1,277 @@
+
+We plan to provide a tool that will still be able to read the bdb databases to
+export the data in ldif
+
+## What are the impact of using lmdb ?
+
+Underlying database implementation is controlled by the nsslapd-backend-implement
+config attribute.
+
+Most functionalities and interfaces are not impacted by switching its value from "bdb" to "mdb" but there are still some impacts:
+
+- Backend configuration attributes
+- Backend monitoring attributes
+- Performances
+
+### Backend configuration attributes impact
+
+Some configuration parameters are used or unused depending of the underlying database:
+
+| nsslapd-backend-implement | Config attribute name                 |
+| ------------------------- | ------------------------------------- |
+| mdb only                  | nsslapd-mdb-max-size                  |
+| mdb only                  | nsslapd-mdb-max-readers               |
+| mdb only                  | nsslapd-mdb-max-dbs                   |
+| bdb only                  | nsslapd-dbcachesize                   |
+| bdb only                  | nsslapd-dbncache                      |
+| bdb only                  | nsslapd-db-logdirectory               |
+| bdb only                  | nsslapd-db-circular-logging           |
+| bdb only                  | nsslapd-db-transaction-wait           |
+| bdb only                  | nsslapd-db-checkpoint-interval        |
+| bdb only                  | nsslapd-db-compactdb-interval         |
+| bdb only                  | nsslapd-db-compactdb-time             |
+| bdb only                  | nsslapd-db-transaction-batch-val      |
+| bdb only                  | nsslapd-db-transaction-batch-min-wait |
+| bdb only                  | nsslapd-db-transaction-batch-max-wait |
+| bdb only                  | nsslapd-db-logbuf-size                |
+| bdb only                  | nsslapd-db-page-size                  |
+| bdb only                  | nsslapd-db-index-page-size            |
+| bdb only                  | nsslapd-db-logfile-size               |
+| bdb only                  | nsslapd-db-trickle-percentage         |
+| bdb only                  | nsslapd-db-spin-count                 |
+| bdb only                  | nsslapd-db-verbose                    |
+| bdb only                  | nsslapd-db-debug                      |
+| bdb only                  | nsslapd-db-locks                      |
+| bdb only                  | nsslapd-db-locks-monitoring-enabled   |
+| bdb only                  | nsslapd-db-locks-monitoring-threshold |
+| bdb only                  | nsslapd-db-locks-monitoring-pause     |
+| bdb only                  | nsslapd-db-named-regions              |
+| bdb only                  | nsslapd-db-private-mem                |
+| bdb only                  | nsslapd-db-private-import-mem         |
+| bdb only                  | nsslapd-db-shm-key                    |
+| bdb only                  | nsslapd-db-cache                      |
```

Comment:
I forgot this one (I thought it was nsslapd-dbcachesize). Not sure to understand what it is for :(

Reply:
It seems to be already  deprecated (no impact except in a log message ... which is IMHO a bug  )